src/10/7-5-3.s

```diff
@@ -0,0 +1,25 @@
+    .text
+    .globl main
+main:
+    LDR     r0, =0x3ff5000
+    LDR     r1, =0x3ff5008
+    LDR     r2, =0x4000
```

Comment:
@DanZambuja Como vc sabe o que vai estar no `0x4000` desse jeito?

Reply:
ele fala no enunciado, que a parada vai estar no 0x4000
pra forma que agt tava simulando nao tem como saber exatamente onde vai estar, mas no caso pra essa placa o dado vai estar nesse endereço

Reply:
a placa só comunica pelo endereço `0x3ff5008`. No caso ele tava falando pra vc ler que ele ta supondo que vc colocou alguma coisa lá antes.

Reply:
Escrita no display de 7 segmentos Displaying the contents of a memory location to the seven-segment display Write ARM assembly to inspect memory location 0x4000. If the location contains a decimal number in the range 0-15, display the contents in hex on the seven-segment LED display. As an example, if 0x4000 contains 14, display an E.
Pelo que entendi desse enunciado, eu leio o que tem no endereco 0x4000 e exponho se for entre 0-15
se eu escrevi algo la, ou se sl alguma outra rotina que aconteceu antes escreveu algo em 0x4000, acho que nao importa muito, ele só fala que o dado que é pra mostrar vai estar em 0x4000

Reply:
Quer que coloque um store de algum dado em 0x4000?
Tipo realmente n sei se é pra eu colocar algo e ler ou assumir que tem alguma coisa ja